R/utilities.R

```diff
@@ -14,9 +14,7 @@
 #' NULL %||% 2
 #'
 #' @export
-`%||%` <- function(x, y) {
-  if (is.null(x)) y else x
-}
+`%||%` <- purrr::`%||%`
```

Comment:
These two are the exact same thing because we had just copied this definition to begin with.